ecommerce-ios/Extensions/View+NavigationBar.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  View+NavigationBar.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 13/05/2021.
+//
+
+import SwiftUI
+
+public extension View {
+
+    func navigationBarLargeTitleItems<T>(trailing: T) -> some View where T: View {
+        overlay(NavigationBarLargeTitleItems(trailing: trailing)
+                    .frame(width: 0, height: 0))
+    }
+}
+
+private struct NavigationBarLargeTitleItems<T: View>: UIViewControllerRepresentable {
```

Comment:
Let's rename to `T` -> `Content`

```suggestion
private struct NavigationBarLargeTitleItems<Content: View>: UIViewControllerRepresentable {
```